earthpy/spatial.py

```diff
@@ -405,7 +417,6 @@ def plot_bands(arr, title=None, cmap="Greys_r",
             ax.set_axis_off()
             ax.set(xticks=[], yticks=[])
         plt.tight_layout()
-        plt.show()
```

Comment:
This call to `plt.show()` seems fairly restrictive. Is it totally necessary? It seems like removing it would allow people to create one plot with multiple arrays, e.g., 

```python
>>> import numpy as np
>>> import earthpy.spatial as es
>>> im = np.random.randint(10, size=(2, 4, 5))
>>> titles = ["Red Band", "Green Band"]
>>> es.plot_bands(im,
...               title=titles,
...               figsize=(12,5),
...               cols=2)
>>> es.plot_bands(im,
...               title=titles,
...               figsize=(12,5),
...               cols=2) 
plt.show()
```

Also, by removing these calls to `plot_bands`, we won't need to worry about interactively closing a bunch of plotting windows when running tests locally. So we get more flexibility for users, and less clicking around for the devs!

Reply:
Solved IRL!